docs/ADDING_MACHINES.md

```diff
@@ -327,12 +327,12 @@ remember that the top, side and bottom textures of a casing must be `modern_indu
 For example:
 ```js
 MIMachineEvents.registerCasings(event => {
-    // Register two casings.
+    // Register a casing.
     // This doesn't register any model! Either add models or add the top/side/bottom textures.
-    event.register("my_fancy_casing", "my_other_casing");
+    event.register("fancy_casing", "Fancy");
 
     // This registers a new casing with the same model as a diamond block!
-    event.registerBlockImitation("my_diamond_casing", "minecraft:diamond_block");
+    event.registerBlockImitation("diamond_casing", "Diamond", "minecraft:diamond_block");
```

Comment:
We can use the display name of the block in this case, no?

Reply:
I agree with this, it threw off my casing code first time around.

Reply:
I'm not seeing a clear solution for this. The english name of the block would need to be pulled from somewhere for datagen. Not sure if that is guaranteed available, especially on a server.